src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift

```diff
@@ -25,8 +25,21 @@ final class SettingsDataDonationViewModel: BaseDataDonationViewModel {
 		var dynamicTableViewModel = DynamicTableViewModel.with {
 			$0.add(
 				.section(
+					header: .image(
+						UIImage(named: "Illustration_Datenspendedatenspende"),
+						accessibilityLabel: AppStrings.NotificationSettings.imageDescriptionOn,
+						accessibilityIdentifier: AccessibilityIdentifiers.NotificationSettings.DeltaOnboarding.imageOn,
+						height: 250
+					),
 					cells: [
-						.headline(text: AppStrings.DataDonation.Info.introductionText),
```

Comment:
Is `AppStrings.DataDonation.Info.introductionText` still used somewhere? if not, we could remove the link in the app strings and perhaps in the localization file, too.

Reply:
it is used in another place but i will confir, im confirming with Kerstin if it should change as well